Feature report (#233)
tdwg/tcs2: docs/feature-report/README.md

# Feature report 

## Why do we need TCS 2?

The Taxon Concept Schema (TCS) is the TDWG standard for exchanging taxonomic
data. It is one of the '2005' group of standards, together with Access to
Biological Collections Data (ABCD) and Structured Descriptive Data (SDD), that
consists of an XML Schema. Unlike ABCD, TCS did not have a dedicated group of
users or maintainers and has not been maintained since shortly after it was
ratified. Any problems that people might have with TCS were never fixed, because
there was no mechanism for making changes to the XML Schema.

As TCS is an XML Schema, data delivered as TCS has to be XML, which was the
predominant format for data exchange at the time. A few years after TCS was
ratified, another format, Comma-delimited Values (CSV), became people's
preferred format for shipping around large amounts of data, at least in the
Biodiversity Data domain. Another issue people have with TCS is that it can only
be used for complete datasets and not for individual Taxon Concepts or Taxon

> Names. 

Reviewer: Hmm. Didn't realize this was the case. Not sure why it is the case. I've often used the RDF version of TCS for single TCs.

Since Darwin Core was ratified in 2009, the Darwin Core Taxon class has been the
main vehicle for shipping around taxonomic data. Darwin Core, because it is a
vocabulary standard, can be delivered in various formats, including CSV.

There has been dissatisfaction with the Darwin Core Taxon class for exchanging
taxonomic data, however, predominantly because people feel it is too permissive.

> There are many different ways to deliver syntactically correct Darwin Core Taxon
> data that works well for the purpose it was designed for, but comes apart when
> people try to incorporate it in systems that are different. The Darwin Core

Reviewer: ```suggestion
Darwin Core Taxon data may be syntactically correct but have a meaning that is incompatible with the receiver's data model. The Darwin Core
```

Taxon class also has references to objects that are not defined and its
implementation allows for data artefacts that are not taxa by any definition,
including that of the Darwin Core Taxon itself.

The latter issues were the reason that the Darwin Core RDF Guide
([\[darwin_core_and_rdfowl_task_groups_darwin_2015\]](../bibliography/#darwin_core_and_rdfowl_task_groups_darwin_2015))
concluded that the creation of functional `dwc:Taxon` instances described using
RDF was not possible at the time of writing and that the task of describing

> taxonomic entities would have to be an effort outside of Darwin Core. This is

Reviewer: ```suggestion
taxonomic entities would have to be an effort outside of Darwin Core. TCS 2 is
```

that effort.

In 2017, the Vocabulary Maintenance Standard (VMS) and Standard Documentation
Standard (SDS) were ratified, which establisheed a mechanism by which existing
TDWG standards should be maintained and documented. In 2018, the then Taxonomic
Names and Concepts Interest Group (TNC)—which was also responsible for the
creation of TCS 1—set out to review TCS, the TDWG Taxon Concept and Taxon Name
LSID Ontologies, the Darwin Core Taxon class and other systems for exchanging
taxonomic data that were out there. In 2020, a new charter for the TNC was
ratified, which made the TNC the Interest Group that maintains TCS and which

> lead to a change of the name of the Interest Group to TCS Maintence Group. A

Reviewer: ```suggestion
lead to a change of the name of the Interest Group to TCS Maintenance Group. A
```

the `accordingTo` property that is required. Every TCS 2 Taxon Concept object
has to have a source, or `accordingTo`, which is a publication or other other
form of communication where a taxon is defined in a certain way. This means that
every Taxon Concept is traceable to a source and identifiable, and that in TCS 2
nominal concepts are not possible. With regards to the Name, in TCS 1 a `Name`
can either be a text node, so a string, or a reference (`ref`) to a `TaxonName`
element with an `id` attribute. This is completely analogous to the use of the
Darwin Core `scientificName` and the TCS `taxonName`, respectively, in TCS 2.

The most important structural change we made was dismantling the Taxon
Relationship object that both TCS 1 and the Taxon Concept Ontology have. We want
TCS to be a vocabulary standard, i.e. a set of terms and definitions, so TCS
should not prescribe a certain syntax. 

Another problem with a relationship object is that it obscures the precise
nature of the relationship. Not all relationship types in TCS 1 are
relationships between Taxon Concepts, some are relationships between Taxon

> Concepts and Taxon Names. Also, relationships between Taxon Names in TCS 1 are

Author: I must have made another change in this part, so I could not just commit this suggestion, but I will work it in manually.

`HasSynonym` is a bit of an unfortunate example, as this is a mixed concept in TCS 1, consisting of synonyms in the strict sense as well as things that are more like the set relationships. I like to think that we split the TCS 1 `has synonym` into `synonym` and `intersects`, so that we can say that `intersects` is not an addition, even though `intersects` can be applied more broadly than `has synonym` (or than I initially thought).

Also, while 'synonym' is indeed a name-to-name relationship, `synonym` in TCS 2 is a concept-to-name relationship, just like `taxonName`

elements (owl:objectProperty in the TDWG Taxon Name LSID Ontology), while
relationships between Taxon Concepts (and some between Taxon Concepts and Taxon
Names) are values in an enumeration (owl:Class in the TDWG Taxon Concept LSID
Ontology).

By elevating the values from the Taxon Relationship Type enumeration to
first-class TCS properties and leaving the syntax out of the standard, people
can choose whether to hang them off the subject Taxon Concept, or use them in a
utility object outside TCS, for example the Darwin Core Resource Relationship
class. The shape of the data may dictate the use of a relationship object, but
the terms have the same meaning, regardless of the syntax.

We currently recognise the following relations between the major objects in TCS:

> ### Relationships between taxa

Author: Yes, will do.
